src/admin-views/app-shop.php

```diff
@@ -2,7 +2,7 @@
 
 	<div class="header">
 		<h1><?php esc_html_e( 'Tribe Event Add-Ons', 'tribe-common' ); ?></h1>
-		<a href="https://theeventscalendar.com/?utm_campaign=in-app&utm_source=addonspage&utm_medium=top-banner" target="_blank"><img src="<?php echo esc_url( tribe_resource_url( 'images/app-shop-banner.jpg', false, 'common' ) ); ?>" /></a>
+		<a href="https://theeventscalendar.com/?utm_campaign=in-app&utm_source=addonspage&utm_medium=top-banner" target="_blank"><img src="<?php echo esc_url( tribe_resource_url( '/images/app-shop-banner.jpg', false, null, $main ) ); ?>" /></a>
```

Comment:
We probably should - I think I left the same feedback in the PR that added those links - but in this case all I'm doing is correcting the resource URL for the images.